Corefile.example

```diff
@@ -1,14 +1,32 @@
 . {
-   debug
-   netbox {
-     token  0123456789abcdef0123456789abcdef01234567
-     url    http://localhost:8000/api/ipam/ip-addresses
-     localCacheDuration 1h
-   }
+  debug
+  netbox {
+    token 0123456789abcdef0123456789abcdef01234567
+    url http://localhost:8000/api/ipam/ip-addresses
+    localCacheDuration 1h
+  }
 
-   hosts {
-        10.0.0.1 example.org
-	10.0.0.3 definitely-not-in-netbox.org
-        fallthrough
-   }
+  hosts {
+    10.0.0.1 example.org
+	  10.0.0.3 definitely-not-in-netbox.org
+    fallthrough
+  }
 }
+
+# The following examples specify what zone(s) should be handled by netbox:
+#
```

Comment:
What is the difference here between the zones?

Reply:
Hopefully the added comments make this clearer also.